index.md

```diff
@@ -308,6 +330,10 @@ Also, and let me quote [Harry Roberts](https://csswizardry.com) once again, **pr
 
 If you ask me, the very first thing a styleguide should do is describe the usual guidelines to write code.
 
+> > his
+>
+> Please consider gender-neutral language here. They/them/their is a perfectly reasonable set of singular gender-neutral pronouns. Little things like this can go a long way towards making a community feel more or less open to women and non-binary people.
```

Comment:
They is plural where 'his, her, he, or she' are singular. For proper non-sexist singular pronouns, you can check out this: http://www.oxforddictionaries.com/words/he-or-she-versus-they

;tldr he => 'he or she' ( english fails again :/ )